docs/sources/datasources/tempo/configure-tempo-data-source.md

```diff
@@ -252,27 +278,37 @@ For example, `${__span.name}`.
 | **\_\_trace.duration** | The duration of the trace.                                                                                                                                                                                                                                                                                                               |
 | **\_\_trace.name**     | The name of the trace.                                                                                                                                                                                                                                                                                                                   |
 
-## Service Graph
+## Additional settings
+
+Use the down-arrow to expand the **Additional settings** section to view these options.
```

Comment:
Hyphen needed?
```suggestion
Use the down-arrow to expand the **Additional settings** section to view these options.
```

Reply:
Nope. I was thinking drop-down instead of down arrow.